homeassistant/components/teslemetry/icons.json

```diff
@@ -88,6 +88,11 @@
         }
       }
     },
+    "number": {
+      "vehicle_state_speed_limit_mode_current_limit_mph": {
```

Comment:
No such entity?

Reply:
I removed these from the PR, obviously missed icons.